packages/cli/src/lib/snapshot/snapshotFactory.ts

```diff
@@ -50,8 +51,31 @@ export class SnapshotFactory {
     return new Snapshot(model, client);
   }
 
-  public static async createFromOrg() {
-    // TODO: need 2 instances of AuthenticatedClient: one for the source org, and the other one for the destination org
+  public static async createFromOrg(
+    resourceTypesToExport: ResourceType[],
+    targetOrg: string
+  ) {
+    const client = await this.getClient(targetOrg);
+    const resourcesToExport = resourceTypesToExport.reduce(
+      (resourceToExport, currentType) => {
+        resourceToExport[currentType] = ['*'];
+        return resourceToExport;
+      },
+      {} as Partial<Record<ResourceType, string[]>>
+    );
```

Comment:
in fine, for this one, I think a for each would be more readable
```suggestion
    let resourcesToExport: Partial<Record<ResourceType, string[]>> = {};
    resourceTypesToExport.forEach(
      (currentType) => {
        resourcesToExport[currentType] = ['*'];
      }
    );
```